client/src/language/languageClient.ts

```diff
@@ -61,10 +61,10 @@ export async function activateLanguageServer (context: ExtensionContext): Promis
 
       // Notify the server about file changes to '.clientrc files contain in the workspace
       fileEvents: [
-        workspace.createFileSystemWatcher('**/*.bbclass', false, true, false),
-        workspace.createFileSystemWatcher('**/*.inc', false, true, false),
-        workspace.createFileSystemWatcher('**/*.bb', false, true, false),
-        workspace.createFileSystemWatcher('**/*.conf', false, true, false)
+        workspace.createFileSystemWatcher('**/*.bbclass', false, false, false),
+        workspace.createFileSystemWatcher('**/*.inc', false, false, false),
```

Comment:
I think I've seen that the watcher event runs rescan-project instead of just parsing. rescan would be too slow for file changes IMO.

Reply:
From my observation, I have seen multiple parses on a single file save but I didn't see it triggered scan.

Reply:
Elinor has had trouble with the watcher events. I think that if it's not properly configured, it will infinetely keep parsing/scanning. On disk changes should not happen in any uncontrolled manner in our workflows. I wonder if we shouldn't just drop any kind of file watch?

Reply:
There is some code for embedded languages in the watch function. If we are worried about the parse/scan, we can remove the function call to parse and figure out something else to trigger it.